core/domain/html_cleaner.py

```diff
@@ -25,6 +25,20 @@
 
 
 def filter_a(name, value):
+    """Filters the target value from the link.
```

Comment:
Returns whether the given attribute of an anchor ('a') tag should be whitelisted.